MaxText/train.py

```diff
@@ -82,6 +83,8 @@ def validate_train_config(config):
   if not config.base_output_directory.startswith("gs://"):
     max_logging.log("WARNING: 'base_output_directory' might be pointing your local file system")
   assert config.steps > 0, "You must set steps or learning_rate_schedule_steps to a positive integer."
+  if "tolerance" in config.__dict__ and (config.tolerance > 1.0 or config.tolerance < 0.0):
```

Comment:
Can you move this check to pyconfig.py? Ideally we keep train.py as small as possible